wiki/osu!_originals/en.md

```diff
@@ -0,0 +1,476 @@
+# osu! originals
+
+**osu! originals** are songs that are specifically made for (or otherwise first premiered in) osu! or other osu!-related events such as tournaments, contests, and [Featured Artist](/wiki/Featured_Artists) announcements. While not all osu! originals are officially licensed and included in the [Featured Artist library](https://osu.ppy.sh/beatmaps/artists), the songs that do are commonly distinguished by the `ORIGINAL` label on their respective artists' pages.
+
+All osu! originals that have been licensed by the game are cleared for use in osu! under non-exclusive terms as per the [osu! music licensing policy](/wiki/Legal/Music_licensing). With the exception of bootleg tracks and certain remixes, other osu! originals generally fall under a similar provision unless otherwise specified.
```

Comment:
i almost forgot about it but i'm pretty sure this last sentence can't really be said to be correct in any way

```suggestion
All osu! originals that have been licensed by the game are cleared for use in osu! under non-exclusive terms as per the [osu! music licensing policy](/wiki/Legal/Music_licensing).
```

Reply:
I think it's something where it's quite safe to assume, but nobody has ever explicitly said it. (Mine are cleared for use, though)

I'm not sure how doable it would be but it might be a good idea to say something along the lines of "probably cleared, but please double check".

Reply:
i think this should be fine

```suggestion
All osu! originals that have been licensed by the game are cleared for use in osu! under non-exclusive terms as per the [osu! music licensing policy](/wiki/Legal/Music_licensing). Conversely, non-licensed osu! originals may require permission for use. See the [Content Usage Guidelines](https://osu.ppy.sh/wiki/en/Rules/Content_Usage_Guidelines) for more information.
```

Reply:
yup, applied